services/event_service.go

```diff
@@ -76,10 +76,6 @@ func (e *EventService) CreateAppEvent(ctx context.Context, newMessage *models.Ev
 		return nil, NewServiceError(http.StatusBadRequest, errors.New("app has no configured endpoints"))
 	}
 
-	if app.IsDisabled {
-		return nil, NewServiceError(http.StatusBadRequest, errors.New("app is disabled, no events were sent"))
-	}
-
```

Comment:
I we should still check this somewhere. If it app is disabled we should save the event and create event deliveries for it, but those event deliveries should not enter the queue.

cc @Subomi

Reply:
That's the current implementation in this PR.